aicsimageio/readers/ome_tiff_reader.py

```diff
@@ -291,10 +286,32 @@ def _read_delayed(self) -> xr.DataArray:
         exceptions.UnsupportedFileFormatError: The file could not be read or is not
             supported.
         """
-        # Create the delayed dask array
-        image_data = self._create_dask_array()
-
-        return self._general_data_array_constructor(image_data)
+        with self.fs.open(self.path) as open_resource:
+            with TiffFile(open_resource) as tiff:
+                # Get unprocessed metadata from tags
+                tiff_tags = self._get_tiff_tags(tiff)
+
+                # Unpack dims and coords from OME
+                dims, coords = self._get_dims_and_coords_from_ome(
+                    ome=self._ome,
+                    scene_index=self.current_scene_index,
+                )
+
+                # Because the data in the TIFF may not have all five OME standard dims
+                # we just use the axes dimension names
+                # We can only do this because OME TIFF axes in tifffile will always
+                # have valid axis names
```

Comment:
when you say valid axis names here, are you including the I and Q names that can come up when tifffile gets confused?

Reply:
So far. I have never encountered an OME-TIFF that when I use `TiffFile.axes` it returns `I` or `Q`. So by "valid" I mean:

"because this is an ome-tiff file we somewhat know that when we call `.axes`, we will only get back OME standard dimensions"

Reply:
I'm just saying it's not guaranteed, which is what the comment says

Reply:
The only reason I commented is that the "always" statement is not true, because in the course of testing OmeTiffWriter, I can construct bad ome tiffs that look relatively innocent but confuse tifffile's reader and result in dims with `I`. I think it would just be more accurate to say "is expected to have" instead of "will always have".

Reply:
Tifffile supports [OME modulo dimensions](https://docs.openmicroscopy.org/ome-model/6.2.2/developers/6d-7d-and-8d-storage.html). Axes may contain 'A' angle, 'P' phase, 'R' tile, 'H' lifetime, 'E' lambda, 'Q' other. I don't think it can contain 'I' unless you are reading a sequence of OME-TIFF files or the file is invalid OME-TIFF.

Reply:
Hmmm after looking at the code again, I have three comments:

1. @toloudis you are absolutely correct that comment should be updated -> see next
2. ... I actually don't think that comment is really needed. All it is really saying to do is "use the axes data from the OME TIFF" which is, well... fine? Nothing special is happening there. What will happen, and this is related to @cgohlke's comment is that if there is extra dimensions in the image that are not a part of our standard set, they will be selected out during the `reshape_data` call. I.E. if this reader does encounter an OME-TIFF with axes of `PQTCZYX` everything will work just as it is, it's just that the reader will only store the `TCZYX` data.
3. @cgohlke that's pretty helpful to know. I have thought about expanding our "DEFAULT_DIMENSIONS_ORDER" to include some of those but until we start seeing images commonly with them probably won't / may support them on the `OmeTiffReader` and not the `AICSImage` object. 🤷 Lot's to discuss basically, and I will probably print it up with Josh Moore / the OME team during some type of community call.